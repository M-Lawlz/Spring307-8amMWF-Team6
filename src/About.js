import React from "react";

export default class About extends React.Component {
  render() {
    return (
      <div>
        <h2>SimTrek: Travel Anywhere, Easily</h2>
        <h3> What is SimTrek? </h3>
        <p>
          SimTrek is a platform that will allow users to access virtual tours
          from the comfort of one’s home. Users will have the ability to create
          a “vtravelr” account and connect with other users, potentially being
          able to take virtual tours together. Furthermore, users will be able
          to rate tours in order to enable user feedback. Virtual tours will
          contain a description of the city, pictures of the city, and a virtual
          tour that is AR accessible.
        </p>
        <h3> New Features Coming Soon </h3>
        <p>
          Profile page for each user Login (allows user virtual tour uploading,
          commenting, saving fav places, etc.) Message other users Forum - where
          tours are uploaded
        </p>
      </div>
    );
  }
}
