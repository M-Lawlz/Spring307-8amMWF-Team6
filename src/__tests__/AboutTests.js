import About from "../About";
import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

test("About renders successfully", () => {
  const about = shallow(<About />);
  expect(toJson(about)).toMatchSnapshot();
});
