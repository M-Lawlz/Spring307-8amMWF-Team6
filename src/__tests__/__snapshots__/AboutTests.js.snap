// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders successfully 1`] = `
<div>
  <h1>
    SimTrek: Travel Anywhere, Easily
  </h1>
  <h2>
     What is SimTrek? 
  </h2>
  <div
    id="theDiv"
  >
    <p>
      SimTrek is a platform that will allow users to access virtual tours from the comfort of one’s home. Users will have the ability to create a “vtravelr” account and connect with other users, potentially being able to take virtual tours together. Furthermore, users will be able to rate tours in order to enable user feedback. Virtual tours will contain a description of the city, pictures of the city, and a virtual tour that is AR accessible.
    </p>
  </div>
  <h2>
     New Features Coming Soon 
  </h2>
  <div
    id="theDiv"
  >
    <p>
      Profile page for each user Login (allows user virtual tour uploading, commenting, saving fav places, etc.) Message other users Forum - where tours are uploaded
    </p>
  </div>
  <h2>
     Our Team! 
  </h2>
  <div
    id="theDiv"
  >
    <p>
      Our founding team consists of Nitish Padavala, Matthew Lawler, and Wilson Huang!
    </p>
  </div>
</div>
`;
